website/blog/2023-01-20-typescript-first.md

```diff
@@ -0,0 +1,101 @@
+
+Flow users have previously upgraded React Native’s Flow types by merging in the `.flowconfig` from the new app template and manually updating `flow-bin`. The new app template no longer has a `.flowconfig`, but [one is still packaged within React Native](https://github.com/facebook/react-native/blob/main/.flowconfig) that can be used as a basis for your app.
+
+If you need to start a new React Native app in Flow, you can reference the [new app template from 0.70](https://github.com/facebook/react-native/tree/0.70-stable/template).
+
+### What if I find a bug in the TypeScript declaration?
+
+Regardless of whether you’re using built-in TS types or `@types/react-native`, if you find a bug please submit a PR to both [React Native](https://github.com/facebook/react-native) and [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) repositories . If you don’t know how to fix it, please file a Github issue in the React Native repository and mention [@lunaleaps](https://github.com/lunaleaps) on the issue.
+
+## Documentation is TypeScript First
+
+To ensure a consistent TypeScript experience, we have made several updates to the React Native documentation to reflect TypeScript as the new default language.
+
+Code examples now allow inline TypeScript and over 170 interactive code examples have been updated to pass linting, formatting, and type-checking in the new template. Most examples are valid as both TypeScript and JavaScript. Where they are incompatible, you can view the example in either language.
+
+If you spot a mistake or you have an improvement, remember that the website is also open source and we would love to see your PRs!
+
+## Thank you to the React Native TypeScript community!
+
+In closing, we want to recognize all the work done over the years by the community to ensure that TypeScript was usable by React Native developers.
+
+We want to thank all the [contributors](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L3) that have been maintaining `@types/react-native` since [2015](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/efce0c25ec532a4651859f10eda49e97a5716a42)! We see the effort and care you all have put into making sure React Native users have the best experience.
+
+Thank you to [@acoates](https://github.com/acoates), [@eps1lon](https://github.com/eps1lon), [kelset](https://github/kelset), [@tido64](https://github.com/tido64), [@Titozzz](https://github.com/Titozzz), and [@ZihanChen-MSFT](https://github.com/ZihanChen-MSFT) for your help consulting, questioning, communicating and reviewing changes to move the TypeScript types to React Native.
+
+Similarly, we want to thank the [maintainers of `ts-template`](https://github.com/react-native-community/react-native-template-typescript/graphs/contributors) for supporting the TypeScript experience for new app development in React Native since day one.
```

Comment:
`ts-template` looks like project's name, while it's `react-native-template-typescript`. I guess we could also refer to it as a "Community TypeScript template", but I prefer the former
```suggestion
Similarly, we want to thank the [maintainers of `react-native-template-typescript`](https://github.com/react-native-community/react-native-template-typescript/graphs/contributors) for supporting the TypeScript experience for new app development in React Native since day one.
```

Reply:
Updated!